examples/stable_diffusion_v2/text_to_image.py

```diff
@@ -413,6 +420,23 @@ def main(args):
         default=None,
         help="path to checkpoint of model",
     )
+    parser.add_argument(
+        "--check_safety",
+        action="store_true",
+        help="set this flag to use a safety checker",
+    )
+    parser.add_argument(
+        "--clip_ckpt_path",
+        type=str,
+        default=None,
+        help="path to checkpoint of clip-vit-large-patch14 for safety checker",
+    )
+    parser.add_argument(
+        "--safety_version",
+        type=int,
+        default=1,
```

Comment:
because we run SD2.0 by default in text_to_image.py, safety_version should be 2 by default

Reply:
done